WordPress/wordcamp.org
# Update wcorg-login-message.php plugin

`wcorg_login_message` returned old WordPress.org login link that referred to forums and logic to create that link was complex. Now that WordPress.org has one centralised "SSO" login at login.wordpress.org, we should direct users there instead of extra redirect hop from old forums login address.

The same message is also used outside `wp-login.php` view, but it didn't contain any link to the place where user can actually do the login and return to the same page. Added this link to the message and made it hidden on `wp-login.php` view, where it is not needed.

At the same time changed the function name from where the wp.org URL is get and deprecated the old function.

Fixes #522 

### Screenshots

**Login prompt on forms with new message and link to wp.org login which will redirect back to the current page.**

![](https://i.imgur.com/75ybu2O.png)

**wp-login.php on WordCamp.org stays as is.**

![](https://i.imgur.com/NVCHdnQ.png)

### How to test the changes in this Pull Request:

1. Navigate to [Call for Speakers post](https://2017.fyre.wordcamp.test/2020/01/26/call-for-speakers/) when not logged in. You should see in message link to do the login.
